File: help/eng_gb/exercicios_da_ficha_9/Ficha09_010/Ficha09_010.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha09_010 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Number : " );
        int num = Integer.valueOf(keyb.nextLine());
        System.out.print("Digits = " );
        while(num > 0 ){
            System.out.print(num % 10 + " " );
            num /=10;
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_010/Ficha09_010.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Number : ");
    int num;
    scanf("%d", &num);
    printf("Digits = ");
    while(num > 0 ){
        printf("%d ",num % 10 );
        num /=10;
    }
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12
